src/world/loaders/UserFixLoader.cpp

```diff
@@ -22,9 +22,13 @@
 
 namespace world {
 
+constexpr const char *USER_REGION = "USER";
+
 UserFixLoader::UserFixLoader(LoadManager *mgr):
     loadMgr(mgr), world(mgr->getWorld())
 {
+    // Create a dummy region for user fixes (they are normally not region coded)
+    world->addRegion(USER_REGION);
 }
```

Comment:
The UserFixLoader creates the dummy region on construction, rather than every time a new fix is added. Potentially there may be many instantiations of UserFixLoader during an Avitab session, but the overhead of re-creating the dummy region will be neglible.